models/org.go

```diff
@@ -125,6 +128,7 @@ func CreateOrganization(org, owner *User) (err error) {
 	org.NumTeams = 1
 	org.NumMembers = 1
 	org.Type = UserTypeOrganization
+	org.Visibility = VisibleType(setting.Service.DefaultVisibilityMode)
```

Comment:
One is not able to set it when it's being created? What if the default is "public" and you want it "private"? 🤔

Reply:
You can still change it after anyway. I do not remember this code. Need deeper look

Reply:
@bkcsoft fixed. User can now chose visibility when creating org.